include/picongpu/fields/incidentField/profiles/InsightPulse.def

```diff
@@ -0,0 +1,122 @@
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#if(ENABLE_OPENPMD == 1) && (SIMDIM == DIM3)
+
+#    pragma once
+
+
+namespace picongpu
+{
+    namespace fields
+    {
+        namespace incidentField
+        {
+            namespace profiles
+            {
+                namespace defaults
+                {
+                    struct InsightPulseParam
+                    {
+                        /** Path to the openPMD input file
+                         *
+                         * It is recommended to use a full path to make it independent of how PIConGPU is launched.
+                         * Relative paths require consistency to the current directory when PIConGPU is started
+                         * with tbg and the standard .tpl files, relative to the resulting simOutput directory.
+                         */
+                        static constexpr char const* filename = "/path/to/file%T.h5";
+
+                        //! Iteration inside the file (only file, not related to the current simulation time iteration)
+                        static constexpr uint32_t iteration = 0;
+
+                        /** Name of the E field dataset inside the input file
+                         *
+                         * Note that only C dataOrder is supported.
+                         *
+                         * @warning it is only the dataset itself, a simple text name and not something like
+                         * "/[directories]/[iteration]/meshes/E".
+                         */
+                        static constexpr char const* datasetEName = "E";
+
+                        /** Datatype of the field record
+                         *
+                         * OpenPMD needs this information at compile time.
+                         */
+                        using dataType = float_64;
+
+                        //! Defining the propagation ( = time) and polarisation axes of the input data
+                        static constexpr char const* polarisationAxisOpenPMD = "x";
+                        static constexpr char const* propagationAxisOpenPMD = "z";
+
+                        /** Unit propagation direction vector in PIConGPU
+                         *
+                         * Must be either X or Y or Z, but can be different from propagationAxisOpenPMD.
+                         *
+                         * Norm of this vector must be 1.0.
+                         * (Normalization is required on a user side as internally it is awkward to do with the
+                         * static-constexpr style of using parameters.)
+                         *
+                         * unit: none
+                         */
+                        static constexpr float_64 DIRECTION_X = 0.0;
+                        static constexpr float_64 DIRECTION_Y = 1.0;
+                        static constexpr float_64 DIRECTION_Z = 0.0;
+
+                        /** Unit E polarization direction in PIConGPU
```

Comment:
In BaseParam.hpp it is spelled "polarization" as well. Since I used some methods from there, I also copied the 'z'; but I will change it to 's' for consistency reasons.